server/routes/pubDocument.tsx

```diff
@@ -112,7 +113,14 @@ const getEnrichedPubData = async ({
 	};
 };
 
-app.get('/pub/:pubSlug/release/:releaseNumber', async (req, res, next) => {
+const speedLimiter = slowDown({
+	windowMs: 60000, // 1 minute for requests to be kept in memory. value of 60000ms is default but expressed here for clarity
+	delayAfter: 240, // allow 240 requests per minute, then...
```

Comment:
I think 240 may be a little high. Maybe 60 requests per minute/one request per second?